paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/EmailConfig.kt

```diff
@@ -32,8 +32,8 @@ internal class EmailConfig : TextFieldConfig {
         return when {
             input.isEmpty() -> Error.Blank
             PATTERN.matcher(input).matches() -> Valid.Limitless
-            containsNameAndDomain(input) -> Error.Invalid()
-            else -> Error.Incomplete
+            containsNameAndDomain(input) -> Error.Invalid(R.string.email_is_invalid)
+            else -> Error.Incomplete(R.string.email_is_invalid)
```

Comment:
This should be incomplete

Reply:
I changed on purpose to align with iOS, it still behaves as `Incomplete` (doesn't show error while it has focus) but the message says "Your email is invalid." when it loses focus, which is correct.